schemas/user/createUserSchema.js

```diff
@@ -35,16 +41,7 @@ module.exports = {
       },
     },
   },
-  emailNull: {
-    custom: {
-      options: async (value, { req }) => {
-        if (!req.body.email) {
-          throw new ErrorObject("email is null");
-        }
-      },
-    },
-  },
-  passwordNull: {
+  password: {
```

Comment:
change the custom field for  password: {
    notEmpty:{errorMessage:"password is null"}